config/boards/sakurapi-rk3308b.csc

```diff
@@ -0,0 +1,17 @@
+# Rockchip RK3308 quad core 256-512MB SoC WiFi
+BOOT_SOC="rk3308"
+BOARD_NAME="Sakura Pi RK3308B"
+BOARD_MAINTAINER="Chiyuki Akatsuki"
```

Comment:
BOARD_MAINTAINER="Chiyuki Akatsuki" -> BOARD_MAINTAINER="TheSnowfield"

If this data is not empty, we need a record [here](https://www.armbian.com/update-data/). Email and name at minimum.

Reply:
Thanks, I have changed the maintainer name and submitted a contact record to Armbian.